intro-to-automated-tests/packages-and-managing-packages.md

```diff
@@ -0,0 +1,134 @@
+
+### Versions Have Significance
+
+Every time a developer wants to update a package they've built and published, they _release_ a new version. Versions are named and referenced usually through versioning numbers.
+
+There are changes in the package's logic in every version. Sometimes these changes are significant-- significant enough that developers who are using that package and depending on it may have compatibility issues.
+
+A worst-case scenario for a developer is broken code that is failing because of a package dependency. Because that is incredibly dangerous, understanding the significance of package versions is important.
+
+**Example:** Arya has released a Python package that calculates how many days until her birthday as version 1.0. Francesca is working on a project that calculates how many days until her half birthday, and it uses Arya's package at version 1.0. If Arya changes her package and erases all of the code, and releases it as version 1.1, what happens to Francesca's project? Francesca's project will fail if it uses version 1.1, but will be okay if it continues to use version 1.0
+
+## Using And Managing Packages With `pip`
+
+[`pip`](https://pypi.org/project/pip/) is a package that is a package installer. With `pip`, we will be able to install any package that is on PyPI.
+
+We will use `pip` to install many dependencies over time.
+
+To use `pip`, we should learn:
+1. How to install it onto our own computers
+1. How to list all the packages installed on our computer
+1. How to install packages
+
+Callout: (IMPORTANT, should be red or something) In the following sections, our commands begin with `pip3`. That may be surprising; why do we use `pip3` instead of `pip`? `pip3` will specify to use `pip` with Python 3, which is not only desired, but **necessary.**
+
+### Installation
+
+Starting with Python 3.4, `pip` gets installed during the Python installation!
+
+To verify that `pip` is installed, we can run our first `pip` command in the terminal:
+
+```bash
+pip3 --version
+```
+
+If there was output that `pip3` is not found, refer to the `pip` docs for an installation guide.
+
+### List All Packages
+
+To list all of the packages and their version numbers installed on our computer, we can use `pip3 list`.
+
+### Install Packages
+
+To install packages using pip, we use
+
+```bash
+pip3 install <packagename>
+```
+
+Where `<packagename>` is replaced with a package name.
+
+Callout: We are largely using `pip` to do global installations on our computer. It is best practice to use `pip` to install packages in a virtual environment; this is out of scope for this curriculum, but feel free to dive deeper.
+
+## Summary
+
+We will find, install, use, and manage packages in Python using `pip`.
+
+Checklist:
+- Define package in Python 
+- Explain that pip is a package installer for Python
+
+Possibly another checklist:
+- install `pip`
+- run `pip3 --version`
+- run `pip3 list`
+
+## Check for Understanding
+
```

Comment:
** will work on this over the weekend **

Reply:
Noooo worries, this is stuff we can get Jared, Kaida, and Chris to do LOLOLOL